infrastructure/cluster/tf/jhub/config.yaml

```diff
@@ -67,15 +67,15 @@ singleuser:
   storage:
     type: none
   image:
-    name: gitlab-registry.cern.ch/escape-wp2/docker-images/datalake-singleuser # replace with updated image in vre-hub
+    name: ghcr.io/goseind/singleuser-base  # replace with updated image in vre-hub
     tag: latest
```

Comment:
despite that we are in testing, still, please DO NOT tag any image with latests.